packages/serializer/addon/-private/transforms/transform.js

```diff
@@ -97,7 +97,9 @@ export default EmberObject.extend({
     @param options hash of options passed to `attr`
     @return The serialized value
   */
-  serialize: null,
+  serialize(deserialized) {
```

Comment:
Actually this was wrong - today, `store.normalize` and `model.serialize` would fail if either `serialize` or `deserialize` methods are missing so just removed those properties from the class and added a unit tests